peachjam/templates/peachjam/layouts/document_list.html

```diff
@@ -19,7 +19,7 @@
     {% endblock %}
 
     {% block page-header %}
-      <div class="row">
+      <div class="row mt-3">
```

Comment:
Would it make sense to instead add margin to the `h1` inside the `page-title` block? 

I think the africanlii page titles for the document lists should have the same margin treatment as for peachjam.

Reply:
done. Used my-4